setup.py

```diff
@@ -1,16 +1,13 @@
 import platform
 from pathlib import Path
 
-from packaging.version import Version
 from setuptools import find_packages
 from setuptools import setup
 
-if Version(platform.python_version()) < Version('3.0.0'):
-    raise Exception("`swagger-ui-py` support python version >= 3.0.0 only.")
-
 setup(
     name='swagger-ui-py',
     version='23.8.23',
+    python_requires='>3.8.0',
```

Comment:
Why  is 3.8？

Reply:
As defined here: https://github.com/PWZER/swagger-ui-py/blob/master/tox.ini#L2

What would you prefer? Everything below 3.8 is EOL https://devguide.python.org/versions/

Reply:
In terms of compatibility, `swagger-ui-py` does not require specified the minimum version of python3, and the tox configuration only specifies individual versions for testing. Although python officially does not support versions before 3.8, we do not need to limit the version.

Reply:
I introduced the line to replace the non-standard (preventing to be able to build this in a normal build environment) code above that checked if version > 3

I am pretty sure (without having tested) that this code will run with a Python Version below 3.7. Which wouldn't matter since one of the dependencies (jinja2) also requires 3.7 as a minimum version. Therefore, I would argue it is better to prevent installation on a python version this code neither compatible with, nor tested on.


But if it is important to you, I can change it to any other value or remove it, which one would you prefer?

Reply:
Okey, I think just use `python_requires='>3.0.0'`. As for python_requires that depends on the library, we don’t need to consider it.